src/main/java/mate/academy/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy.bookstore.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.order.external.ShippingAddressRequestDto;
+import mate.academy.bookstore.dto.order.external.StatusRequestDto;
+import mate.academy.bookstore.dto.order.internal.OrderResponseDto;
+import mate.academy.bookstore.dto.orderitem.internal.OrderItemDto;
+import mate.academy.bookstore.service.order.OrderService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/orders")
+@Tag(name = "Order management",
+        description = "Endpoints for managing orders and items in it")
+public class OrderController {
+    private final OrderService orderService;
+
+    @PostMapping
+    @Operation(summary = "Create a new order",
+            description = "Creation a new order based on the user's shopping cart and address")
+    public void createOrder(
+            Authentication authentication,
+            @RequestBody ShippingAddressRequestDto requestDto
```

Comment:
Consider validation here
```suggestion
            @RequestBody ShippingAddressRequestDto requestDto
```